src/components/sbb-expansion-panel-header/sbb-expansion-panel-header.tsx

```diff
@@ -72,7 +72,9 @@ export class SbbExpansionPanelHeader implements ButtonProperties, ComponentInter
 
   private _onMouseMovement(toggleDataAttribute: boolean): void {
     const parent: HTMLSbbExpansionPanelElement = this._element.closest('sbb-expansion-panel');
-    if (parent) {
+    // The `sbb.hover-mq` logic has been removed from scss, but it must be replicated to have
+    // the correct behavior on mobile. Android devices need also `(pointer: fine)`.
+    if (parent && window.matchMedia('(any-hover: hover) and (pointer: fine)').matches) {
```

Comment:
About first question, I checkout-ed the commit before the hover change ([2710e74](https://github.com/lyne-design-system/lyne-components/pull/1860/commits/2710e744b665f1fd174e4f449e0e04c1aa304b06)) and I see on Android the hover state applied on click; so in theory, 'pointer: fine' should be applied, but, for consistency, I can remove it.

About second one, I agree, I could change it (with or without 'pointer: fine') to:
```
if (!toggleDataAttribute || (parent && window.matchMedia('(any-hover: hover) and (pointer: fine)').matches)) {
  toggleDatasetEntry(parent, 'toggleHover', toggleDataAttribute);
}
```

Reply:
I like your commit :). For the pointer: fine, maybe we then would have to enhance also the scss code with the new logic. But this would probably imply much much more testing for every use case :D So I can live with the current implementation ;) (without :fine)